ssh/DOCS.md

```diff
@@ -6,50 +6,57 @@ Follow these steps to get the add-on installed on your system:
 
 1. This addon is only visible to "Advanced Mode" users. To enable advanced mode, go to **Profile** -> and turn on **Advanced Mode**.
 2. Navigate in your Home Assistant frontend to **Supervisor** -> **Add-on Store**.
-3. Find the "SSH server" add-on and click it.
+3. Find the "Terminal & SSH" add-on and click it.
 4. Click on the "INSTALL" button.
 
-## How to use
-
-You need enable the port for external access. You can just enter 22 as value or any other
-value like you want. This recommend to add login credentials.
-
-To use this add-on, you must have a private/public key to log in.
-To generate them, follow the [instructions for Windows][keygen-windows]
-and [these for other platforms][keygen]. It is possible to set a password for
-login since version 2.0 but for high security use private/public keys.
 
-You can not run both variants at the same time. Enabling login via keys, will
-disable password login.
+## How to use
 
-1. Add a ssh key to  `authorized_keys` or set a `password` in the add-on configuration.
-2. Start the add-on.
-3. Connect to your device using your preferred SSH client and use `root` as
-   the username.
+Once installed (following the steps above) you need to
+* enable external access by selecting which TCP port to open up (see Network section), and
+* supply login credentials,
+* start the add-on.
```

Comment:
This is now incorrect. You don't have to use SSH. Furthermore, external access can be confused with opening ports up in a router.

We use dashes for lists (`-`) in Markdown and please start items in a list with a capital letter.

Reply:
I can certainly adapt to use `-` in Markdown bullet lists! I searched for, but didn't find a style guide for documentation (or use of Markdown features) in this project. I looked here: https://developers.home-assistant.io/ and here: https://developers.home-assistant.io/docs/development_guidelines (and somehow missed the top menu item Documentation).

As I failed to find a style guide for docs, I went with what most references online use, which is `*`. Example references (top two results when I use Google to search for markdown bullet list): 
- https://www.markdownguide.org/basic-syntax/#adding-elements-in-lists
- https://guides.github.com/features/mastering-markdown/

I now spent some more time, and see that https://developers.home-assistant.io/docs/documenting/ exists, there's no recommendation on what should be preferred. In fact, the only reference I can find to Markdown is this: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code which suggests any use of `*-+`, and lists them in that order with `*` being the first one used to demonstrate bullet lists.

I'm happy to make a change (PR) in https://developers.home-assistant.io/docs/documenting/standards/ if other that want to contribute to keeping documentation up-to-date will get remarks about this undocumented preferred style.

Reply:
In response to your first statement:

I didn't realize the in-browser Terminal could be enabled from the current documentation. As I learned this now, I more or less rewrote the whole section to take this into account. While doing so, the text now nudges towards that one should only enable the SSH Server/Client part "if you know what you're doing".

Reply:
@frenck - this section was more or less completely rewritten. I believe the issues you raised are (better) addressed with the new text. Are we OK to close this?